ubo_app/services/090-assistant/ubo-service/pyproject.toml

```diff
@@ -9,7 +9,7 @@ readme = "README.md"
 requires-python = ">=3.11"
```

Comment:
I think this commit can be merged with the previous commit which is also related to assistant pipeline. It would make sense to have it as a separate commit if we need to do extra work to make it work or if we did it intentionally to add a feature, fix an issue, etc.